wp-content/themes/csisnuclearnetwork/assets/_scss/components/_share.scss

```diff
@@ -39,32 +34,32 @@
     }
   }
 
-  /* stylelint-disable-next-line */
-  .a2a_button_email {
+  /* stylelint-disable */
+  .a2a_button_facebook {
     position: relative;
-    top: 2px;
+    top: -2px;
   }
 
+  .a2a_button_linkedin {
+    position: relative;
+    top: -2px;
+  }
+
+  /* stylelint-enable */
   .single__footer &,
   .single__content & {
-    --icon-fill: invert(31%) sepia(62%) saturate(350%) hue-rotate(131deg) brightness(94%) contrast(92%);
-
-    --icon-fill-hover: invert(0%) sepia(40%) saturate(1505%) hue-rotate(213deg) brightness(117%) contrast(108%) #{$shadow--1};
+    --icon-fill: invert(13%) sepia(100%) saturate(2787%) hue-rotate(189deg) brightness(98%) contrast(101%);
 
-    @include breakpoint('small') {
-      width: fit-content;
-      margin-right: auto;
-      margin-left: auto;
-    }
+    --icon-fill-hover: invert(14%) sepia(10%) saturate(7274%) hue-rotate(181deg) brightness(95%) contrast(87%) #{$shadow--1};
 
     &::before {
       content: 'Share';
       margin-right: rem(12);
       font-weight: 600;
       text-transform: uppercase;
-      opacity: 0.8;
+      opacity: 0.7;
       filter: var(--icon-fill);
```

Comment:
The label is a different color than the icons, so we should use a different filter and remove the opacity.

Reply:
Nice catch! I added the correct color but we usually have an opacity on this of 0.8. It's supposed to look a little more faded than the icons. Now that the filter is correct - can you take one more look?